model/src/main/java/com/microsoft/java/bs/gradle/model/GradleSourceSet.java

```diff
@@ -24,11 +24,6 @@ public interface GradleSourceSet extends Serializable {
    */
   public String getDisplayName();
 
-  /**
-   * Equivalent to {@code org.gradle.api.Project.getName()}. 
-   */
-  public String getProjectName();
```

Comment:
The project name is used by JDT.LS when creating a new `IProject` instance during project import. Let's preserve it.

Reply:
I've reinstated project name